test/gtest/uct/uct_test.h

```diff
@@ -332,47 +331,38 @@ class uct_test : public testing::TestWithParam<const resource*>,
     void wait_for_bits(FlagType *flag, MaskType mask,
                        double timeout = DEFAULT_TIMEOUT_SEC) const
     {
-        ucs_time_t deadline = ucs_get_time() +
-                              ucs_time_from_sec(timeout) *
-                              ucs::test_time_multiplier();
+        using ucs::get_deadline;
+        ucs_time_t deadline = get_deadline(timeout);
         while ((ucs_get_time() < deadline) && (!ucs_test_all_flags(*flag, mask))) {
             /* Don't do short_progress_loop() to avoid extra timings */
             progress_mt();
         }
     }
 
-    template <typename T>
-    void wait_for_value(volatile T *var, T value, bool progress,
+    template<typename T>
```

Comment:
minor: space removed here needs to be readded

Reply:
according to code style checker (clang-format) it should without space